integrations/oci/oci-secrets-config-source/src/test/java/io/helidon/integrations/oci/secrets/configsource/UsageTest.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.integrations.oci.secrets.configsource;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import io.helidon.config.Config;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.not;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+class UsageTest {
+
+    private UsageTest() {
+        super();
+    }
+
+    @Test
+    void testUsage() {
```

Comment:
i'm still not seeing the name change or other pure unit test coverage here.

Reply:
Could you give me an idea of what you would like to see tested? And what the name should be?

Reply:
There is some 400+ lines in `OciSecretsConfigSourceProvider` that should be tested since as it is right now there is nothing currently test covered in the normal case where the pipeline or developer typically does not have an $HOME/.oci/config present in their environment.  I am suggesting we have some unit-test coverage on this code for the typical case.  Perhaps even consider making more methods package private to be able to test them in isolation, eg. ` static boolean isModified(Instant pollInstant, Instant closestSecretExpirationInstant)`, `static void completeAllSecretsTasks(Collection<? extends Callable<Void>> tasks, AutoCloseable secrets)`, ` Optional<NodeContent> load(Supplier<? extends Vaults> vaultsSupplier,
                                           Supplier<? extends Secrets> secretsSupplier,
                                           ListSecretsRequest listSecretsRequest)` with some fakes or mocks, etc.